pkg/cnb/buildpack_validation.go

```diff
@@ -16,7 +17,9 @@ func (bl BuildpackLayerInfo) supports(buildpackApis []string, id string, mixins
 	}
 
 	for _, s := range bl.Stacks {
-		if s.ID == id {
+		iaAnystack := s.ID == "*" && api.MustParse(bl.API).Compare(api.MustParse("0.5")) >= 0
```

Comment:
MustParse will panic on a not parseable input which is actually an underlying user input. `api.NewVersion` is probably safer.

Reply:
We already pull in semver to parse buildpack versions. Could we use that instead of adding another dependency on the lifecycle?

Reply:
ya good call